sdks/cpp-lite/common/include/DeviceAccessor.h

```diff
@@ -0,0 +1,17 @@
+
+#pragma once
+
+#include "Param.h"
+#include <example.tiny.json.h>
```

Comment:
This include statement would need to change based on the name of the generated file.